src/loc/ContextualizedLocDetails.tsx

```diff
@@ -287,6 +288,9 @@ export default function ContextualizedLocDetails(props: Props) {
                 locId={ locId }
                 loc={ loc }
             />
+            { loc.locType === 'Collection' &&
+                <CollectionLocItemChecker locId={ locId } />
```

Comment:
Should be displayed only once collection LOC is closed (see [this comment](https://github.com/logion-network/logion-internal/issues/347#issuecomment-1038892385) from Elie).

Reply:
Done.